aws/resource_aws_instance.go

```diff
@@ -639,7 +639,9 @@ func resourceAwsInstance() *schema.Resource {
 		CustomizeDiff: customdiff.All(
 			SetTagsDiff,
 			func(_ context.Context, diff *schema.ResourceDiff, meta interface{}) error {
-				if diff.HasChange("launch_template.0.version") {
+				_, ok := diff.GetOk("launch_template")
+
+				if diff.Id() != "" && diff.HasChange("launch_template.0.version") && ok {
```

Comment:
Apparently `HasChange` with an empty state will always return true when resource filed has `Default` value, so I've added a check for top-level key.